src/context.js

```diff
@@ -1,26 +1,51 @@
 // @flow
 import React, { type Node } from 'react';
+import LocalStorage from './storages/local';
 
 // $FlowFixMe
 const { Provider, Consumer } = React.createContext({ data: {}, setResponse: () => {} });
 
 type ProviderProps = {
-  children: Node
+  children: Node,
+  storagePrefix: string
 };
 type ProviderState = {
   data: Object
 };
+type ResponsePair = {
+  response: any,
+  error: any
+};
+
+const DefaultStoragePrefix = 'react_loads_';
 
 class LoadsProvider extends React.Component<ProviderProps, ProviderState> {
+  static defaultProps = { storagePrefix: DefaultStoragePrefix };
```

Comment:
This definitely looks like a type declaration lol

Reply:
Wym?

Reply:
coz it's pascal case